Shear/AppDelegate.swift

```diff
@@ -16,6 +16,9 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     var app: App!
 
     func applicationDidFinishLaunching(_ aNotification: Notification) {
+        
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/c3bc09a3e7d5aea4d33d4de57a16cb964bae126d/Shear/AppDelegate.swift#L20"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AppDelegate.swift:20</code></del></td>
    </tr>
  </tbody>
</table>